google/ggrc-core
# GGRC-7393/GGRC-7394 Add person collection to assessment template

# Issue description

While adding local custom attribute with Person type in Assessment template, it should be the possibility to add person collection to this LCA with Person type on Assessment's Info pane.

# Steps to test the changes

1. Have program, control, audit
2. Open Audit page> Assessment template
3. While creating AT add local CA w/Person type and Save
4. Generate Assessment based on Control and AT
5. Navigate to Assessment's Info pane and try to add several person into local CA with person type

Actual Result: There is a possibility to add only one person into local CA with person type
Expected Result: Add person collection with the possibility to make multiple selection

# Solution description

Add `attribute_objects` attribute to cav. Extend unique constraint on `custom_attribute_values` with `attribute_object_id`. Update `_api_attrs` with attribute_objects that relates to persons who mapped to ca_value.

Since we have several cavs to one cad we must provide equality of cads and cavs, so we also change return order for `custom_attribute_values` when json is generated for object.

Also resolve 409 error on FE side

# Sanity checklist


# Migration checklist

# PR Review checklist
